apps/heureka/src/components/filters/FilterSelect.jsx

```diff
@@ -50,8 +50,9 @@ const FilterSelect = ({ entityName, isLoading, filterLabels, filterLabelValues,
           disabled={!filterLabelValues[filterLabel]?.length}
           className="filter-value-select w-96 bg-theme-background-lvl-0"
         >
-          {filterLabelValues[filterLabel] //Ensure already selected values are not displayed in filterValue drop down to avoid duplicate selections
-            ?.filter((value) => !activeFilters[filterLabel]?.includes(value)) // Filter out values that are already active
+          {/* Ensure already selected values are not displayed in filterValue drop down */}
```

Comment:
I would consider here also to extract the logic to variable and remove the comments from the return statement. Something like this:
```
const filteredOptions = filterLabelValues?.[filterLabel] // Safely access filterLabelValues
  ?.filter((value) => !activeFilters?.[filterLabel]?.includes(value)); // Safely access activeFilters and filter out already active values
```

and then use it in the return statement:

```
    {filteredOptions?.map((value) => (
      <SelectOption value={value} key={value} />
    ))}
```